newIDE/app/package.json

```diff
@@ -100,6 +101,8 @@
     "extract-all-translations": "node scripts/extract-all-translations.js",
     "compile-translations": "node scripts/compile-translations.js",
     "reload-extensions": "node scripts/import-GDJS-Runtime.js",
+    "build-theme-resources": "node scripts/compile-theme-styles.js",
+    "new-theme": "node scripts/create-new-theme.js",
```

Comment:
Nitpicking: I usually prefix all commands by a verb, much like a function: "create-new-theme", for explicitness.

Reply:
done ✅